.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Feature request
+about: Suggest an idea or a feature for this project
+title: ''
+labels: feature request
+assignees: ''
+---
+
+<!--
+**Please do not report security vulnerabilities here**. The [Responsible Disclosure Program](https://auth0.com/whitehat) details the procedure for disclosing security issues.
+
+**Thank you in advance for helping us to improve this library!** Your attention to detail here is greatly appreciated and will help us respond as quickly as possible. For general support or usage questions, use the [Auth0 Community](https://community.auth0.com/) or [Auth0 Support](https://support.auth0.com/). Finally, to avoid duplicates, please search existing Issues before submitting one here.
```

Comment:
I'm hesitant to use this markdown formatting, as being inside an HTML comment it will never be rendered on the final view. I'd put links without the md syntax. Or even leave those out of this comment, as are now going to be part of the "issue selector" view.

Reply:
Best to raise this with the team on the global template doc I think, since this would apply to all repos.